README.de.md

```diff
@@ -0,0 +1,51 @@
+# Sway
+Sway ist ein mit i3 kompatibler [Wayland](http://wayland.freedesktop.org/) compositor. Bei fragen lese die [FAQ](https://github.com/swaywm/sway/wiki). Oder trete dem [IRC Channel](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway on irc.freenode.net) bei (Englisch).
+
+Falls du die Entwicklung von Sway unterstützen möchtest, kannst du das auf [SirCmpwn's Patreon Seite](https://patreon.com/sircmpwn) machen.
+
+## Signaturen
+Alle Releases sind mit [B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A) signiert und werden auf GitHub veröffentlicht.
+
+## Installation
+### Mit dem Paket Manager
+Sway kann in vielen Distributionen direkt durch den Paket Manager installiert werden. Das Paket sollte "sway" heißen, falls es kein solches Paket gibt, kannst du auf der [Wiki](https://github.com/swaywm/sway/wiki/Unsupported-packages) (englisch) nach mehr Informationen suchen.
+
+Falls du sway für deine eigene Distribution als Paket bereitstellen möchtest, solltest du die Entwickler auf IRC oder per E-Mail (sir@cmpwn.com) kontaktieren.
+
+### Selber Kompilieren
+
+Installiere die folgenden Bibliotheken
+
+* meson\*
+* [wlroots](https://github.com/swaywm/wlroots)
+* wayland
+* wayland-protocols\*
+* pcre
+* json-c
+* pango
+* cairo
+* gdk-pixbuf2 (Optional wird für den System tray benötigt)
+* [scdoc](https://git.sr.ht/~sircmpwn/scdoc) (Optional wird für die man Hilfeseiten benötigt)
+* git\*
+
+_\*Werden nur während des Kompilierens benötigt_
+
+Führe die folgenden Befehle aus
+
+    meson build
+    ninja -C build
+    sudo ninja -C build install
+
+Falls dein System kein logind hat, musst du noch der sway Datai die passenden Berechtigungen geben:
```

Comment:
```suggestion
Falls dein System nicht logind benutzt, musst du sway noch die passenden Berechtigungen geben:
```

Reply:
This is way better. I mean my sentence got the point across, but I wasn't really happy with it.